src/stactools/landsat/utils.py

```diff
@@ -225,3 +228,37 @@ def get_usgs_geometry(
         return item.geometry
     else:
         return None
+
+
+def handle_antimeridian(item: Item, antimeridian_strategy: Strategy) -> None:
+    """Handles some quirks of the antimeridian.
+
+    Applies the requested SPLIT or NORMALIZE strategy via the stactools
+    antimeridian utility. If the geometry is already SPLIT (a MultiPolygon,
+    which can occur when using USGS geometry), a merged polygon with different
+    longitude signs is created to match the expected input of the fix_item
+    function.
+
+    Args:
+        item (Item): STAC Item
+        antimeridian_strategy (Antimeridian): Either split on +/-180 or
+            normalize geometries so all longitudes are either positive or
+            negative.
+    """
+    geometry = shape(item.geometry)
+    if isinstance(geometry, MultiPolygon):
+        # force all positive lons so we can merge on an antimeridian split
+        polys = list(geometry.geoms)
+        for index, poly in enumerate(polys):
+            if poly.centroid.x < 0:
+                polys[index] = shapely.affinity.translate(poly, xoff=+360)
```

Comment:
Just because the centroid is > 0 doesn't mean there aren't negative longitudes, correct? Don't you need to check _all_ longitudes to see if they're < 0?

Reply:
udpated.